literatur.bib

```diff
@@ -12144,6 +12200,15 @@ @InProceedings{muise-et-al-icaps2022systemdemos
   crossref =     "icaps2022systemdemos"
 }
 
+@Article{muise-et-al-jair2016,
+  author =       "Christian Muise and J. Christopher Beck and Sheila A. McIlraith",
+  title =        "Optimal partial-order plan relaxation via {M}ax{SAT}",
```

Comment:
Using {...} like this is not strictly speaking wrong, but I would write it as {MaxSAT} instead. Same result, less syntax and works much more nicely with spellcheckers.